Debugger/Main.as

```diff
@@ -93,92 +77,593 @@ namespace VehicleDebugger
 
 	void TabAllStates()
 	{
-		auto sceneVis = GetApp().GameScene;
-		if (sceneVis is null) {
-			UI::Text("Not currently in a scene");
+		auto App = GetApp();
+		if (App.GameScene is null) {
+			UI::Text("\\$F00Not currently in a scene");
 			return;
 		}
 
-		auto allVis = VehicleState::GetAllVis(sceneVis);
+		auto allVis = VehicleState::GetAllVis(App.GameScene);
 		for (uint i = 0; i < allVis.Length; i++) {
-			auto vis = allVis[i];
-			uint entityId = VehicleState::GetEntityId(vis);
+			auto Vis = allVis[i];
+			uint entityId = VehicleState::GetEntityId(Vis);
```

Comment:
Is there a reason you changed the capitalization on all these variables? (Makes the diff bigger which feels a bit unnecessary)